src/libcore/iter.rs

```diff
@@ -8,51 +8,278 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Composable external iterators
+//! Composable external iteratoration
 //!
-//! # The `Iterator` trait
+//! If you've found yourself with a collection of some kind, and needed to perform an operation on
```

Comment:
Iterators also come from things like streams, but it'd be best to leave that for later I think.